Add basic introduction web page (#4)
icure/medtech-docs: sdks/intro.md

# Introduction

## iCure for Medtech

> iCure is a secured and distributed healthcare platform that helps you create apps and share data with the healthcare community.

Reviewer: ```suggestion
*iCure* is a secured and distributed healthcare platform that helps you create apps and share data with the healthcare community.
```

> iCure for medtech has been tailor-made for medical device manufacturers and the development teams responsible for writing software for medical devices.

Reviewer: ```suggestion
*iCure for medtech* is tailored for medical device manufacturers and for the development teams responsible for the software of medical devices.
```

> Data gathered from medical devices are secured when collected and stored encrypted in our/your infrastructure.

Reviewer: ```suggestion
*iCure* allows you to secure the data gathered from medical devices upon collection, and to store it encrypted in our cloud or on your premises.
```

> Simple to use SDKs allow developers to focus on creating appealing apps while letting us handle the details of making their app secure.  

Reviewer: ```suggestion
Our simple to use SDKs allow developers to focus on creating appealing apps while we handle the complex security aspects of their application.  
```

## Supported development platforms

iCure Medtech can be used in the browser or with Node.

> Experimental versions of the SDKs are available for React Native, Flutter and native Android/iOS development and can be requested by contacting us.

Reviewer: ```suggestion
We are also working on experimental versions of the SDKs for React Native, Flutter and native Android/iOS development. You can request these experimental SDKs by contacting us.
```
